updates/2022_07_07_contracts_block_timestamp_update.ts

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable */
+import { connect } from '../src/db/connect';
+import axios from 'axios';
+import { Benchmark } from 'redstone-smartweave';
+async function updateContractsWithBlockTimestamp() {
+  require('dotenv').config({
+    path: '.secrets/local.env',
+  });
+
+  const db = connect();
+
+  const blockHeights: any = await db.raw(`SELECT DISTINCT block_height FROM contracts ORDER BY block_height ASC;`);
```

Comment:
I could select the height up to the height from which we are already updating but I could add it after these changes are installed https://github.com/warp-contracts/gateway/pull/78. Although, I believe the script is quite fast so I would not pay that much attention to it.